examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb

```diff
@@ -97,7 +97,11 @@
     "from dask_saturn import SaturnCluster\n",
```

Comment:
<p>it doesn't look like you changed any of the values. How did this get past the linting checks and onto <code>main</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>im not sure actually...i ran <code>make format</code>  before commiting</p>

Reply:
ok no prob

Reply:
<p>wait if <code>make format</code>  is going to change this, then you should just leave it with whatever <code>make format</code>  decided. It's weird to me that <code>format</code>  and <code>lint</code>  didn't agree, but we also shouldn't have to manually revert this every time we touch the repo.</p>